src/Type/BaseDoctrineORMSerializationType.php

```diff
@@ -125,6 +125,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 continue;
             }
 
+            // @phpstan-ignore-next-line
```

Comment:
This seems better in the baseline.

Phpstan report useless baseline but does not report useless phpstan-ignore.
This way it will be reported as an error when the baseline wont be needed (since i fixed the phpdoc in the propertyMetadata class)

Reply:
👍 Fixed.